nvm.sh

```diff
@@ -1883,8 +1883,7 @@ nvm_install_source() {
     ./configure --prefix="${VERSION_PATH}" $ADDITIONAL_PARAMETERS && \
     $make -j "${NVM_MAKE_JOBS}" ${MAKE_CXX-} && \
     command rm -f "${VERSION_PATH}" 2>/dev/null && \
-    $make -j "${NVM_MAKE_JOBS}" ${MAKE_CXX-} install && \
-    command rm -rf "${TMPDIR}"
```

Comment:
I think I'd still like to remove the dir if the installation fails. Can we move this down to line 1906?